template/11-monetization.md

```diff
@@ -1,14 +1,18 @@
 # Business Model
 
-*Expected operating Costs*
+## Revenue Streams
 
-- Development costs  
-- Infrastructure costs: hosting, storage, and bandwidth  
-- Maintenance costs  
-- Marketing costs  
-- Operational costs: customer Support, administration, legal and compliance  
+Our business model for Proxima is mainly based on advertising for businesses of any sizes. Our main value proposition is providing localized-based content, allowing businesses of any size to reach their targeted audience. This enables a campaign of any size based on the live location of the user, offering from worldwide coverage to precise targeting down to districts or even neighborhoods for centralized actions. This can be viewed as a digital version of local billboards.
 
-*Revenue Streams*
+Our localized based content is also able to provide event organizers (such as festival organizers) time-limited event experiences for users within certain predefined areas. This is aimed to be beneficial to the three parties interacting. Our value proposition is to provide participant retention and event discovery through our heat map. We can engage the participants through challenges, point-earning and enlarged post reach during the event. This fosters application discovery and user loyalty.
+
+Proxima's business model is based on the density of users interacting in specific areas. We have to scale it according to how our user base grows in each region. We plan to launch at EPFL, and expand to new places, progressively allowing close monitoring and implementation of this business model.
+
+
+## Expected operating Costs
+
+At launch, Proxima's marking costs will be our primary expenses, as we depend on users' density within the launch area to grow.
+
+Once Proxima starts expanding, we expect marketing cost to grow with it to help kick start user acquision within new area. This grow will be accompanied with operational costs, providing customer and user support, and getting in line with local laws. We also project advertiser acquisition to take up a considerable part of our operating costs to increase our revenue streams.
+As for other social media, infrastructure costs such as hosting, bandwidth, and storage of user interactions and posts make up a significant part of our operating costs. However, our model offers significant advantages, allowing the storage of posts near their physical presence and reducing the cost of it. We believe that development, along with research and development, will take a moderate part of our expensive to keep our application relevant and increase user engagement.
```

Comment:
```suggestion
Just like other social media, infrastructure costs such as hosting, bandwidth, and storage of user interactions and posts make up a significant part of our operating costs. However, our model offers significant advantages, allowing the storage of posts near their physical presence and reducing the cost of it. We believe that development, along with research and development, will take a moderate part of our expenses to keep our application relevant and increase user engagement.
```

Reply:
Thank you ! Fixed in: [832b24e](https://github.com/ProximaEPFL/prd/pull/6/commits/832b24ebd009e02d2ab4815159b72894790952e1)